doc/website/blog/2023-12-12-hibernate-schema-loader.mdx

```diff
@@ -0,0 +1,598 @@
+    public MovieParticipationKey(Long movieId, String actorName) {
+        this.movieId = movieId;
+        this.actorName = actorName;
+    }
+    public Long movieId;
+
+    public String actorName;
+}
+```
+
+</TabItem>
+<TabItem value="Main.java">
+
+```java
+package org.example;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.boot.MetadataSources;
+
+import java.util.Set;
+
+public class Main {
+    public static void main(String[] args) {
+        SessionFactory sessionFactory = new MetadataSources()
+                .addAnnotatedClass(Movie.class)
+                .addAnnotatedClass(Actor.class)
+                .addAnnotatedClass(MovieParticipation.class)
+                .buildMetadata()
+                .buildSessionFactory();
+
+        sessionFactory.inTransaction(session -> {
+                List<Actor> actors = session.createQuery("from Actor").list();
+                if (!actors.isEmpty()) {
+                actors.forEach(x -> System.out.println("Found Actor " + x.name));
+            } else {
+                Movie matrix = new Movie("The Matrix", 1);
+                Actor keanuReeves = new Actor("Keanu Reeves");
+                keanuReeves.actedIn = Set.of(new MovieParticipation(matrix, keanuReeves));
+                session.persist(keanuReeves);
+            }
+        });
+    }
+}
+```
+
+</TabItem>
+</Tabs>
+
+:::note
+Currently, Atlas does not support using generated fields that require data initialization such as GenerationType.SEQUENCE, GenerationType.TABLE, and Generation.AUTO.
+
+If needed, you can still export the schema using the flag --enable-table-generators (or -Denable-table-generators using Maven), when applying the schema to your database, you will need
+to make sure to apply the ignored statements (using `atlas migrate --env hibernate diff --edit`, see more information on manual migrations [here](/versioned/new))
+
+For example, if you are adding GenerationType.SEQUENCE to the Event entity, you will to add an insert statements to your generated migration file:
+
+```diff
+diff --git a/migrations/20231210140844.sql b/examples/with_local_plugin_repository/migrations/20231210140844.sql
+index ad80a64..5955834 100644
+--- a/migrations/20231210140844.sql
++++ b/migrations/20231210140844.sql
+@@ -4,3 +4,6 @@ CREATE TABLE `Event` (`id` bigint NOT NULL AUTO_INCREMENT, `title` varchar(255)
+ -- Create "Event_SEQ" table
+ CREATE TABLE `Event_SEQ` (`next_val` bigint NULL) CHARSET utf8mb4 COLLATE utf8mb4_0900_ai_ci;
++ -- Initialize "Event_SEQ" table
++ insert into Event_SEQ values ( 1 );
+```
+
+Testing these changes can be done by running the application with a local database and creating the entity. To apply the migration directory
+to the local database, use `atlas migrate apply`.
+
+We would love to hear your feedback :heart:. You can reach out to us on [Discord](https://discord.gg/zZ6sWVg6NT) or by opening an [issue](https://github.com/ariga/atlas-provider-hibernate/issues/new).
```

Comment:
Feels foreign to put this here again in the note

Reply:
which part? just the part about the feedback or also the note about testing?